pg_search/sql/pg_search--0.15.8--0.15.9.sql

```diff
@@ -0,0 +1,13 @@
+\echo Use "ALTER EXTENSION pg_search UPDATE TO '0.15.9'" to load this file. \quit
+
+/* <begin connected objects> */
+-- pg_search/src/api/index.rs:771
+-- pg_search::api::index::term
+CREATE  FUNCTION "term"(
+	"field" FieldName, /* pg_search::api::index::FieldName */
+	"value" inet DEFAULT NULL /* core::option::Option<pgrx::datum::inet::Inet> */
+) RETURNS SearchQueryInput /* pg_search::query::SearchQueryInput */
+IMMUTABLE PARALLEL SAFE 
+LANGUAGE c /* Rust */
+AS 'MODULE_PATHNAME', 'inet_wrapper';
+/* </end connected objects> */
```

Comment:
Taken from `cargo pgrx schema`. It would be better if that script is generated by the CI and added as a comment instead of the current output where each action is a single line thing without comments.